crates/sui-core/src/authority.rs

```diff
@@ -2173,17 +2187,25 @@ impl AuthorityState {
                             .into())
                         }
                         Some(object) => {
-                            let layout = object.get_layout(
-                                ObjectFormatOptions::default(),
-                                // threading the epoch_store through this API does not
-                                // seem possible, so we just read it from the state (self) and fetch
-                                // the module cache out of it.
-                                // Notice that no matter what module cache we get things
-                                // should work
-                                self.load_epoch_store_one_call_per_task()
-                                    .module_cache()
-                                    .as_ref(),
-                            )?;
+                            let layout = {
```

Comment:
The removed comment seems to still match the code - was it removed as redundant wrt. the code?

Reply:
I think we are fine but thanks for pointing it out.
I added that comment a while ago and I think it's not critical and things are fine as they are anyway